### ISSUE TITLE
Precalculate ancestor uuids to help frontend build breadcrumb

### ISSUE THREAD

**Issue by vnugent** [OpenBeta/openbeta-graphql#21]
Assuming area page has the following slug:  `areas/<uuid>/optional-area-name`

The breadcrumb component will need all ancestor uuids + display names 

```
      {
        "area_name": "Smith Rock",
        "children": [
          {
            "area_name": "Aggro Gully",
            "metadata": {
              "lat": 44.36724,
              "lng": -121.14238
            }
          },
          _path: {
              [ <uuid of great grandparent>, "Oregon"],
              [<grandparent uuid>, "Central Oregon"],
              [<parent uuid>,"Smith Rock"],
           }
       ...
        ]
      }
```
<img width="706" alt="Screen Shot 2021-12-30 at 11 53 16 AM" src="https://user-images.githubusercontent.com/3805254/147776665-04c74e69-ff52-448a-926f-eedd13d7388d.png">

**Comment by gibboj** [comment]
https://github.com/OpenBeta/openbeta-graphql/pull/25 I implemented this for areas before I saw this ticket so I did it a little differently. `pathTokens` I stored & the `ancestors` as a list of `area_id`, I added as a resolver. 

I think I could improve this PR by : 
1. removing pathTokens or pathHash, seems like duplicate information. We don't need to store them together. 
    a. Maybe `ancestors` is the information we store in area instead, then use the resolver to get the name. Then get rid of `pathHash` and `pathTokens`
2. Matching the data in the resolver. Right now ancestors has no direct connection to the string. I just depend on them being in hierarchal order.

**Comment by vnugent** [comment]
While working on the new db seeding code (https://github.com/OpenBeta/openbeta-graphql/pull/31) I realize we're not really using pathHash for anything else other than the initial area-climbing linking.  I'm going to remove it from db & graphql schemas.

**Comment by vnugent** [comment]
fixed in #31
